templates/tableau-cluster-linux-master.template

```diff
@@ -160,11 +161,20 @@
                 },
                 "QSS3KeyPrefix": {
                     "default": "Quick Start S3 Key Prefix"
+                },
+                "AcceptEULA": {
+                    "default" : "Accept Tableau End User License Agreement"
```

Comment:
Make this a smaller label like "Accept EULA". Additional details go in Description of this parameter. Otherwise, the UI will look really overloaded on the label side of things.

Reply:
I checked UI. I think the length of the label is ok. A customer doesn't know what EULA is.
Besides, "Tableau Services Manager (TSM) administrator username" has longer time.